tracer/build/_build/UpdateVendors/VendoredDependency.cs

```diff
@@ -203,27 +216,30 @@ private bool TryGetValue(string key, [NotNullWhen(true)]out JsonProperty? item)
                                 @$"using\s+(\S+)\s+=\s+{Regex.Escape(originalNamespace)}.(.*);",
                                 match => $"using {match.Groups[1].Value} = Datadog.Trace.Vendors.{originalNamespace}.{match.Groups[2].Value};");
 
-                        
-                        
+
+
                         // Don't expose anything we don't intend to
                         // by replacing all "public" access modifiers with "internal"
                         return Regex.Replace(
                             result,
-                            @"public(\s+((abstract|sealed|static|unsafe)\s+)*?(partial\s+)?(partial\s+)?(class|readonly\s+struct|struct|interface|enum|delegate))",
+                            @"public(\s+((abstract|sealed|static|unsafe)\s+)*?(partial\s+)?(partial\s+)?(class|readonly\s+ref\s+struct|readonly\s+struct|struct|interface|enum|delegate))",
```

Comment:
This regex doesn't look right to me 🤔?

- `(partial\s+)?(partial\s+)?` - can there ever be two partial keywords? (I realise this was already there)
- `ref\s+struct|readonly\s+` I think the new addition should just be `(ref\s+)?`? But then I think `readonly` should be optional too, so 🤔 
```suggestion
                            @"public(\s+((abstract|sealed|static|unsafe)\s+)*?(partial\s+)?(class|readonly\s+(ref\s+)?struct|struct|interface|enum|delegate))",
```